model_hub/model_hub/mmdetection/_trial.py

```diff
@@ -61,16 +62,15 @@ def __init__(self, context: det_torch.PyTorchTrialContext) -> None:
         # We will control how data is moved to GPU.
         self.context.experimental.disable_auto_to_device()
 
+        dist.init_process_group(backend="gloo")
```

Comment:
can we `init_process_group` as part of the controller's pre execute hook similar to what we do for deepspeed and enable users to override?
https://github.com/determined-ai/determined/blob/master/harness/determined/pytorch/deepspeed/_deepspeed_trial.py#L70

Reply:
we should probably default to `nccl` backend instead of `gloo`

Reply:
done